parm/config/config.base.emc.dyn

```diff
@@ -187,6 +190,10 @@ case "${APP}" in
   ATM)
     export confignamevarfornems="atm"
     ;;
+  ATML)
+    export DO_LAND="YES"
```

Comment:
where will this variable `DO_LAND` be used?

Reply:
It will be used in the `workflow/applications.py` in the future PRs.

Reply:
I'm still not clear on the utility of land being off. Adding as a new app for ATM implies we will also need a "+land" option for all the other apps as well. Is that a road we want to go down?

Reply:
@WalterKolczynski-NOAA I added the `DO_LAND=YES` to define the `ATML` app for doing the land DA, and this followed the `DO_AERO=YES` to define the `ATMA` app by @CoryMartin-NOAA. 

In the `workflow/applications.py`, the land DA app is added by `DO_LAND` as below: 
```
        self.do_land = _base.get('DO_LAND', False)

        if self.do_land:
            configs += ['landanlinit', 'landanlrun', 'landanlfinal']
        if self.do_land:
            gdas_gfs_common_tasks_before_fcst += ['landanlinit', 'landanlrun', 'landanlfinal']
```
@CoryMartin-NOAA Would you please reply to questions by @WalterKolczynski-NOAA ? Thanks.

Reply:
Hi @jiaruidong2017, @WalterKolczynski-NOAA is right, as we discussed a few weeks ago with Mike B., there will never be a ATML app, since land is a required part of the ATM model. Perhaps we need an option to only run the land DA for certain members, but let's tackle that at a later date.

Reply:
@CoryMartin-NOAA Do you have any specific suggestions on how to proceed with this? Thanks.

Reply:
we should talk about it with Clara and Mike but I'm thinking we don't make it optional with `do_land` because by definition there will always be 'land' with 'atm'